src/Scrollable/Scrollable.test.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react'; // (or /dom, /vue, ...)
+import useIsVisible from '../hooks/useIsVisible';
+
+// adds special assertions like toHaveTextContent
+import '@testing-library/jest-dom/extend-expect';
+
+import Scrollable, { CLASSNAME_SCROLL_BOTTOM, CLASSNAME_SCROLL_TOP } from './index';
+
+jest.mock('../hooks/useIsVisible');
+
+function rangeOfNumbers() {
+  return Array.from({ length: 50 }, (v, k) => k + 1);
+}
+
+function renderScrollable(items) {
+  render(
+    <div style={{ display: 'flex', height: '5vh' }}>
```

Comment:
If this display: flex is required I think it should be part of `<Scrollable />`.

Reply:
yeah so if I remove it and leave my padding in the scrollable body it still sort of works.. I guess I don't need it here

so making

```
.pgn__scrollable-body {
    overflow: auto;
    position: relative;
```

then removing the flex here also works, but it's edge detection now is a bit less smooth :(

Reply:
removed padding from scrollable stlying and display flex too